lib/commands/init.js

```diff
@@ -107,17 +107,17 @@ module.exports = Command.extend({
         }
       })
       .then(function() {
-        if (!commandOptions.skipNpm) {
-          return npmInstall.run({
-            verbose: commandOptions.verbose,
-            optional: false
+        if (!commandOptions.skipBower) {
+          return bowerInstall.run({
+            verbose: commandOptions.verbose
```

Comment:
what do you mean with "installed packages" check? you mean that the postinstall runs immediately after you create a new project? If so: isn't it a good thing to create the cache right away?